api/src/paths/taxonomy/taxon/tsn/hierarchy.ts

```diff
@@ -0,0 +1,106 @@
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { getAPIUserDBConnection } from '../../../../database/db';
+import { ItisService } from '../../../../services/itis-service';
+import { getLogger } from '../../../../utils/logger';
+
+const defaultLog = getLogger('paths/taxonomy/taxon/tsn/hierarchy');
+
+export const GET: Operation = [getTaxonHierarchyByTSN()];
+
+GET.apiDoc = {
+  description: 'Get taxon records by TSN ids.',
+  tags: ['taxon_id'],
+  security: [
+    {
+      Bearer: []
+    }
```

Comment:
The other PR removes the security requirements for the previous 2 taxon endpoints. Should this also have no security?

Reply:
Yes, thanks for catching that